content/data_science/3_introduction_to_statistics/introduction_to_statistics/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+_db_id: 106
```

Comment:
Frontmatter is configuration. This is code. You need to work intentionally. You need to work with precision.

_db_id should NEVER be handcoded. It is the database id of a piece of content in our database. What you are doing here is trying to overwrite another piece of content in our database. Please, if you don't know what something doers do not guess. Find out.

The content_type is missing. Is this a topic or a project?

Look at other topics and projects and try to understand the frontmatter. Then make sure you always make the frontmatter complete.  Don't add extra things that should not be there. And make sure you fill in all the required fields.

Read the contribute.md file inside this repo. A lot of what I am telling you has already been written down. Please make sure you read the docs.